<launch>
  <include file="$(find ar_track_alvar)/launch/pr2_indiv_no_kinect.launch"/>

  <node name="final_docking_algorithm" pkg="docking" type="final_docking_algorithm" output="screen">
    <param name="zError" type="double" value="0.03"/>
    <param name="ReverseDistance" type="double" value="0.65"/>
    <param name="CheckPointCount" type="int" value="30"/>
    <remap from="ar_pose_marker" to="ar_pose_marker_Small"/>
  </node>

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node"/>

  <node name="image_view" pkg="image_view" type="image_view" args="image:=usb_cam/image_raw">
      <param name="/usb_cam/pixel_format" value="yuyv"/>
  </node>

</launch>
